content/en/docs/tasks/administer-cluster/reserve-compute-resources.md

```diff
@@ -87,7 +87,7 @@ kubernetes system daemons.
 To optionally enforce `kube-reserved` on kubernetes system daemons, specify the parent
 control group for kube daemons as the value for `--kube-reserved-cgroup` kubelet
 flag.  If `kube-reserved` is enforced, kubelet will evict processes 
-of the specified cgroup if the thresholds are exceeded.
+of the specified cgroup if a threshold is exceeded.
```

Comment:
"if the threshold is exceeded" is more accurate?

Reply:
I updated the text (twice)